### PR TITLE
[Issue-1841] Wipe command gate on user prompt

### PR DESCRIPTION
**Pull request by shahvineet98**
The wipe commands are pretty dangerous commands. A verification step allows the user to type `DELETE` to confirm they want to execute the command. I have applied this to both the run and schedule commands where wipe can be run. I also updated the tests respectively. This PR is based on what was mentioned in #1841 

**Before:** 
image

**After:**
image

image

### CODE REVIEW

File: python_modules/dagster/dagster/cli/run.py [hunk 1]
```diff
@@ -20,6 +20,10 @@ def run_list_command():
 
 @click.command(name='wipe', help='Eliminate all run history. Warning: Cannot be undone')
 def run_wipe_command():
-    instance = DagsterInstance.get()
-    instance.wipe()
-    click.echo('Deleted all run history')
+    confirmation = click.prompt('Are you sure you want to delete all run history? Type DELETE')
```
Comment (mgasner) [inline python_modules/dagster/dagster/cli/run.py]: maybe a little aggressive vs. `y/N`

Comment (mgasner) [reply python_modules/dagster/dagster/cli/run.py]: should also read "all run history and event logs?"

File: python_modules/dagster/dagster/cli/schedule.py [hunk 2]
```diff
@@ -390,7 +390,12 @@ def execute_wipe_command(cli_args, print_fn):
...
-    scheduler.wipe()
-
-    print_fn("Wiped all schedules")
+    confirmation = click.prompt(
+        'Are you sure you want to delete all schedules and crontabs? Type DELETE'
+    )
+    if confirmation == 'DELETE':
+        scheduler = schedule_handle.get_scheduler()
+        scheduler.wipe()
+        print_fn("Wiped all schedules")
```
Comment (mgasner) [inline python_modules/dagster/dagster/cli/schedule.py]: use `click.echo`

Comment (helloworld) [reply python_modules/dagster/dagster/cli/schedule.py]: `print_fn` == `click.echo` here

Comment (mgasner) [reply python_modules/dagster/dagster/cli/schedule.py]: hm, why just here?

Comment (shahvineet98) [reply python_modules/dagster/dagster/cli/schedule.py]: I used `print_fn` because this is what was used in other functions in this file

**Pull request merged**